runtime/domain.c

```diff
@@ -891,7 +918,9 @@ CAMLprim value caml_domain_spawn(value callback, value mutex)
 #endif
 
   if (err) {
-    caml_failwith("failed to create domain thread");
+    caml_failwith("failed to create domain thread. "
```

Comment:
This change is incorrect, and should be reverted. The failure to create the thread is not related to `Max_domains` parameter. This comes later.